tests/__snapshots__/test_help_messages.ambr

```diff
@@ -251,48 +251,61 @@
-  |                                                   specific record type. Can  |
-  |                                                   be specified multiple      |
-  |                                                   times.                     |
-  | --scope                    TEXT                   Restrict results to a      |
-  |                                                   specific scope name. Can   |
-  |                                                   be specified multiple      |
-  |                                                   times.                     |
-  | --first                    INTEGER                Fetch only the first N     |
-  |                                                   events. Cannot be used     |
-  |                                                   with --last.               |
-  | --last                     INTEGER                Fetch only the last N      |
-  |                                                   events. Cannot be used     |
-  |                                                   with --first.              |
-  | --follow           -f                             Continue polling for       |
-  |                                                   events. Implies --last 20  |
-  |                                                   unless overridden or the   |
-  |                                                   --since flag is used.      |
-  | --follow-interval          INTEGER                Polling interval in        |
-  |                                                   seconds when using the     |
-  |                                                   --follow flag.             |
-  |                                                   [default: 10]              |
-  | --project          -p      TEXT                   Path where Snowflake       |
-  |                                                   project resides. Defaults  |
-  |                                                   to current working         |
-  |                                                   directory.                 |
-  | --env                      TEXT                   String in format of        |
-  |                                                   key=value. Overrides       |
-  |                                                   variables from env section |
-  |                                                   used for templating.       |
-  | --help             -h                             Show this message and      |
-  |                                                   exit.                      |
+   By default, this command will fetch events generated by an app installed in    
+   the current connection's account. To fetch events generated by an app          
+   installed in a consumer account, use the --consumer-org and --consumer-account 
+   flags. This requires event sharing to be set up to route events to the         
```

Comment:
nit: flags kind of implies it's a boolean... maybe we use arguments or parameters instead?

Reply:
updated to "options" since that's what the `--help` uses
![image](https://github.com/user-attachments/assets/d80fcd59-6281-4667-abd8-e2c53d8e9a2d)